src/test/resources/buildlist-1.3.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
it seems like a good test for delete/getbuild is to make sure veracodeAPI.delete and veracodeAPI.getBuildList get executed and it could return mock response so we can have an example of what we would expect from veracodeAPI?